File: 02/aux.py
from solve import solve, solve_wo_div


def main():
    for index, array in enumerate(get_tests()):
        print(
            "Test %d\nFor array %s\nSolution with division: %s\nSolution without division: %s\n\n" %
            (index, array, solve(array), solve_wo_div(array))
        )


def get_tests():
    return [
        [1, 2, 3, 4, 5],
        [2, 3, 4],
        [0, 2, 3, 4]
    ]


if __name__ == '__main__':
    main()

File: 09/aux.go
package main

import "fmt"

func main() {
	for index, test := range getTests() {
		fmt.Printf("Test %d\nFor array %v test, solution: %d\n\n", index, test, solve(test))
	}
}

func getTests() [][]int {
	return [][]int{
		{5, 1, 1, 5},
		{2, 4, 6, 2, 5},
		{-1, 2, 3, -2, -15, -60, 60, 65, -58, 12, 45, 12, 12, 12, 0, 0, 44},
		{-1, -1},
		{0, 1, 2, 3, 4, 5, -12},
		{0, 1, 4, 3, 4, 7, -12},
	}
}
